### DISCUSSION TITLE
✅ Unable to get Port Forwarding to Work

### DISCUSSION THREAD

**Discussion by rsclmumbai** [isontheline/pro.webssh.net#623] [General]
My web server is running on a remote server listening on 127.0.0.1
I connect to this remote server via SSH + Local port 80 Fwd to Remote 80.
Works fine on my laptop using Putty.  Putty settings here: https://ibb.co/xS2yFy3
I make an SSh connection via Putty & open http://127.0.0.1 on my laptop's browser. Works well.

I'm trying to get the same setup done on my IPad.
My Settings are WebSSH > TUNNEL > {SSH Connection Details} > Port Forwarding = **80:localhost:80**

I'm able to connect fine but when I open Safari or Chrome and try to access https://127.0.0.1 I'm unable to access the application

Is there anything I'm missing?

Thanks for your help!

**Comment by isontheline** [comment]
Hello @rsclmumbai 👋

You need to enable « VPN-Over-SSH » setting inside your forward settings on WebSSH.

It will allow you to use the forward outside WebSSH.

**Comment by rsclmumbai** [comment]
@isontheline Thanks for a quick reply.
It worked perfectly. 
Thanks again!!!!

**Reply by isontheline** [reply]
You’re welcome 

Have a nice day ☀️
